build/Makefile.openssl

```diff
@@ -11,6 +11,6 @@ $(INSTALL_DIR)/bin/openssl: Makefile
 
 Makefile: config Configure Makefile.org
 	# Force a 64 bit build on MacOS
-	KERNEL_BITS=64 ./config --install_prefix=$(INSTALL_DIR) --prefix=/ --libdir=lib --openssldir=$(OPENSSLDIR) no-shared enable-static-engine -fPIC
+	KERNEL_BITS=64 ./config --install_prefix=$(INSTALL_DIR) --prefix=/ --libdir=lib --openssldir=$(OPENSSLDIR) no-asm no-shared enable-static-engine -fPIC
```

Comment:
Should we try to contain this to Travis builds?
(See https://github.com/google/certificate-transparency/blob/master/build/Makefile.glog for a possible approach.)

Reply:
Yeah I suppose we should.

Reply:
I think that should do it.